DataStructure/Array/Arrayhomework1.php

```diff
@@ -0,0 +1,64 @@
+<?php
+$a = [5, 12, 17, 9, 3];
+$b = [13, 4, 8, 14, 1];
+$c = [9, 5, 3, 7, 21];
+
```

Comment:
In homework, it's 3 difference inputs, they are just and examples. You only need to implement for one array.
In the case you want to check with other inputs, you should make a function with argument is an array, and can can re-use this function to check other input like arrays $a, $b, $c above.